### PR TITLE
[terra-application-navigation] Drawer Menu not closing on outside click in IE

### LINKED ISSUE #1006: [terra-application-navigation] Drawer menu will not dismiss with outside clicks in Internet Explorer
# Bug Report

## Description

At small viewports, the ApplicationNavigation component renders a drawer menu that is presented upon selection of the 'hamburger' icon within the navigation header. This menu is dismissed a number of ways: by selecting items within the menu, by pressing the escape key, and by clicking anywhere outside the drawer menu.

For IE10 and IE11, clicking outside the drawer menu will not dismiss. Item selection/Escape key works fine.

## Steps to Reproduce

1. Navigate to engineering.cerner.com/terra-core within IE
2. Shrink the browser window until the compact navigation layout is rendered.
3. Click the hamburger icon to present the drawer menu.
4. Click outside the drawer menu and be disappointed when the menu doesn't dismiss

## Expected Behavior

The drawer menu should be dismissed with outside clicks consistently across all supported browsers.

## Possible Solution

A transparent overlay is rendered over the layout's body contents while the drawer menu is open. I assume click events on that element are firing or being handled in these specific browsers.

## Environment

* Component Name and Version: terra-application-navigation, current
* Browser Name and Version: IE10, IE11
* Operating System and version (desktop or mobile): Desktop->Windows

### PR DESCRIPTION
**Pull request by lokesh-0813**
In smaller view ports when Drawer Menu is open, clicking outside of Drawer Menu does not close the drawer. This issue is happening only in IE. This PR fixes the issue.

fixes #1006

### CODE REVIEW

File: packages/terra-application-navigation/src/ApplicationNavigation.jsx [hunk 1]
```diff
@@ -417,24 +428,19 @@ const ApplicationNavigation = ({
...
-          {children}
-        </main>
-        <Overlay
-          isOpen={drawerMenuIsOpen}
-          isRelativeToContainer
-          backgroundStyle="clear"
-          zIndex="6000"
-          onRequestClose={generateFocusToggle(handleRequestClose)}
-        />
+        <OverlayContainer className={cx('overlay-layout')} overlay={addOverlay()}>
```
Comment (tbiethman) [inline packages/terra-application-navigation/src/ApplicationNavigation.jsx]: @lokesh-0813 Using the OverlayContainer here will start executing the `inert` logic that the Overlay uses to make the content behind the overlay inaccessible. This shouldn't be necessary, as we're using a FocusTrap for the DrawerMenu to ensure focus is trapped within the menu when open. I'm not sure if/how the addition of `inert` will change the overall accessibility of ApplicationNavigation as a whole, so I'm a little wary of the change.

I did notice that the Overlay element in this scenario was positioned offscreen. Notice the vertical offset in this screenshot:

Screen Shot 2020-02-06 at 3 05 10 PM

I'm wondering if just setting a `top: 0px` style on the Overlay will fix this?

Comment (lokesh-0813) [reply packages/terra-application-navigation/src/ApplicationNavigation.jsx]: Hey @tbiethman , `top:0px` fixed the issue. I have updated here d1e0a651d4377e21498877b109fcda39d83854f1

File: packages/terra-application-navigation/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ Changelog
 Unreleased
 ----------
 ### Fixed
+* Fixed Drawer menu not closing in Internet Explorer on clicking outside of drawer menu.
+
+### Changed
+* Updated `content-layout` background color.
```
Comment (supreethmr) [inline packages/terra-application-navigation/CHANGELOG.md]: has `content-layout` background color changed....??

Comment (tbiethman) [reply packages/terra-application-navigation/CHANGELOG.md]: This is from a change that has since been reverted. We should remove this.

Comment (lokesh-0813) [reply packages/terra-application-navigation/CHANGELOG.md]: Removed here 17a0ea9012a2a5806429e85c6939eb0a35cdbbdb and here d6e207cf53358ec57af360c2959bc6588b27b481

**Pull request merged**